src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordWithOcrTest.java

```diff
@@ -231,13 +304,70 @@ private ValidatableResponse postWithBody(byte[] body) {
             .then();
     }
 
-    private static Map<String, Object> document(String filename, String documentNumber) {
-        return ImmutableMap.of(
-            "value", ImmutableMap.of(
-                "fileName", filename,
-                "controlNumber", documentNumber,
-                "someNumber", 3
-            )
+    private CaseDetails exceptionRecord(String attachToCaseReference) {
+        return exceptionRecord(
+            attachToCaseReference,
+            null,
+            null,
+            CASE_TYPE_EXCEPTION_RECORD,
+            EXCEPTION_RECORD_DOC,
+            false
         );
     }
+
+    private CaseDetails exceptionRecord(
+        String attachToCaseReference,
+        String searchCaseReferenceType,
+        String searchCaseReference,
+        String caseTypeId,
+        Map<String, Object> document,
+        boolean containsPayment
+    ) {
+        return CaseDetails.builder()
+            .jurisdiction(JURISDICTION)
+            .id(EXCEPTION_RECORD_ID)
+            .caseTypeId(caseTypeId)
+            .data(
+                exceptionDataWithDoc(
+                    document,
+                    attachToCaseReference,
+                    searchCaseReferenceType,
+                    searchCaseReference,
+                    containsPayment
+                )
+            ).build();
+    }
+
+    private Map<String, Object> exceptionDataWithDoc(
+        Map<String, Object> scannedDocument,
+        String attachToCaseReference,
+        String searchCaseReferenceType,
+        String searchCaseReference,
+        boolean containsPayment
+    ) {
+        Map<String, Object> exceptionData =
+            Maps.newHashMap("scannedDocuments", ImmutableList.of(scannedDocument));
+
+        if (attachToCaseReference != null) {
+            exceptionData.put("attachToCaseReference", attachToCaseReference);
+        }
+
+        if (searchCaseReferenceType != null) {
+            exceptionData.put("searchCaseReferenceType", searchCaseReferenceType);
+        }
+
+        if (searchCaseReference != null) {
+            exceptionData.put("searchCaseReference", searchCaseReference);
+        }
+
+        if (containsPayment) {
+            exceptionData.put("containsPayments", "Yes");
+            exceptionData.put(ExceptionRecordFields.ENVELOPE_ID, "21321931312-32121-312112");
+            exceptionData.put(ExceptionRecordFields.PO_BOX_JURISDICTION, "sample jurisdiction");
+        }
+
+        exceptionData.put("journeyClassification", "SUPPLEMENTARY_EVIDENCE");
```

Comment:
Probably doesn't matter here, but it looks like the appropriate classification for this test class would be `SUPPLEMENTARY_EVIDENCE_WITH_OCR`.